public/index.html

```diff
@@ -2,14 +2,14 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="icon" href="%PUBLIC_URL%/paw.png" />
```

Comment:
Thank you for adding these changes! Looks great!

Reply:
One thing I'm noticing is that it doesn't show up super well in dark mode

![image](https://user-images.githubusercontent.com/46770903/155863491-f31f56c0-21a6-4df2-9f48-8e2f0913b7da.png)

I don't think it's a big deal. Maybe we could change the color to the same one we're using as our header? Totally optional

Reply:
thank you!